userAuth.js

```diff
@@ -81,6 +103,18 @@ router.get('/auth/redirect', passport.authenticate(authStrategy, {failureRedirec
   res.redirect(req.session.authRedirect || formatUrl('/'))
 })
 
+router.get('/metadata', function (req, res) {
+  if (isTouchstoneSamlAuth) {
+    res.type('application/xml')
+    res.send((touchstoneSamlStrategy.generateServiceProviderMetadata(
+      process.env.TOUCHSTONE_SAML_MD_DECRYPTION_CERT,
```

Comment:
What is this decrypting?

Reply:
Mike mentioned that his also used decryption cert which you're ensured & verify a secure transaction between IDP and service.  https://mitodl.slack.com/archives/G5RHT8GDD/p1690560282038369?thread_ts=1690556534.378059&cid=G5RHT8GDD for reference, here's Mike's mention. If we find it's unnecessary I can take it and the related PK out, but if there's a decrypt PK, the xml is expected to carry it as well.

also, from docs: https://github.com/node-saml/passport-saml#generateserviceprovidermetadata-decryptioncert-signingcert-

Reply:
Yeah I get that, but guess my question was more around the wording itself. That's the cert that the IDP uses from the SP to encrypt the assertion and then the SP uses the private key to decrypt. Should we call it `TOUCHSTONE_SAML_SP_ENCRYPTION_CERT` instead?